.vscode/launch.json

```diff
@@ -8,24 +8,41 @@
       "type": "node",
       "request": "launch",
       "name": "Firestore Unit Tests (Node)",
-      "program": "${workspaceRoot}/node_modules/gulp/bin/gulp.js",
-      "args": ["test", "--suite=firestore/unit/", "--env=node"],
-      "port": 9229,
+      "program": "${workspaceRoot}/packages/firestore/node_modules/.bin/_mocha",
+      "cwd": "${workspaceRoot}/packages/firestore",
```

Comment:
FWIW- We could probably move this whole .vscode directory under packages/firestore (each package can have its own .vscode if it wants it)...

Reply:
Sorry, that wasn't very clear.  I was thinking that we'd push people towards opening vscode inside of a specific package.  So if I'm doing firestore development, I'd open vscode in packages/firestore and then our launch configuration would be available and only our sources would show up in vscode, etc.  I think this might make sense, since this launch.json is 100% Firestore-specific anyway...  But I don't feel strongly about this. We can always do it later.